README.md

```diff
@@ -219,6 +219,47 @@ let () = printGithubRepos ()
 ## Node-specific
 
 #### Read lines from a text file
+
+```ml
+let () =
+    Node_fs.readFileSync "README.md" `utf8
```

Comment:
The bucklescript style recommendation is to access these functions via `Node.Fs.readFileSync` rather than the underscore, in case the internal module tree changes.